Keas.Mvc/Controllers/FeedController.cs

```diff
@@ -8,18 +8,21 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 using Keas.Core.Domain;
+using Keas.Mvc.Services;
 
 namespace Keas.Mvc.Controllers
 {
     [AllowAnonymous]
     public class FeedController : SuperController
     {
          private readonly ApplicationDbContext _context;
+         private readonly IReportService _reportService;
 
-        public FeedController(ApplicationDbContext context)
-        {
-            this._context = context;
-        }
+         public FeedController(ApplicationDbContext context, IReportService reportService)
+         {
+             this._context = context;
+             _reportService = reportService;
```

Comment:
why use `this._context` one line up but not `this._reportService` on this line?

Reply:
Don't need the this. I'll remove that in both places 
Blame @jcubbage  :)